cli/command/utils.go

```diff
@@ -16,3 +26,62 @@ func extractArgs(received []string, names ...string) (map[string]string, error)
 
 	return args, nil
 }
+
+func buildLogQueryHelper(id, start, end string, tail int) url.Values {
+	query := url.Values{}
+
+	if tail > 0 {
+		query.Set(client.LogQueryParamTail, strconv.Itoa(tail))
+	}
+
+	if start != "" {
+		query.Set(client.LogQueryParamStart, start)
+	}
+
+	if end != "" {
+		query.Set(client.LogQueryParamEnd, end)
+	}
+
+	return query
+}
+
+func SetTimeMultiplier(v time.Duration) func() {
```

Comment:
don't think we need a time multiplier in this package after moving the helper to `client`

Reply:
✅